db/00000000000001_initial_schema.up.sql

```diff
@@ -151,7 +151,7 @@ CREATE TABLE task_upload_counters(
 
     CONSTRAINT fk_task_id FOREIGN KEY(task_id) REFERENCES tasks(id) ON DELETE CASCADE,
     CONSTRAINT task_upload_counters_unique UNIQUE(task_id, ord)
-);
+) WITH (fillfactor = 50);
```

Comment:
Let's copy the comment from https://github.com/divviup/janus/blob/release/0.6/db/00000000000004_task_upload_counters_fillfactor.up.sql explaining the `fillfactor` value here.